src/main/java/demo/TmScore.java

```diff
@@ -0,0 +1,44 @@
+package demo;
+
+import java.util.List;
+
+import org.apache.spark.api.java.function.Function;
+import org.apache.spark.broadcast.Broadcast;
+import org.biojava.spark.function.TmScorer;
+import org.rcsb.mmtf.spark.data.Segment;
+
+import scala.Tuple2;
+import scala.Tuple3;
+
+/**
+ * Mapper function to perform TMScore analysis.
+ * @author Anthony Bradley
+ *
+ */
+public class TmScore implements Function<Tuple2<Integer,Integer>,Tuple3<Long, Long, Double>> {
+
+	/**
+	 * The serial version of this commit.
+	 */
+	private static final long serialVersionUID = -2557285958500330806L;
+	private Broadcast<List<Tuple2<String, Segment>>> calphaChains;
```

Comment:
This means that every node needs to keep a copy of all data in memory, right? I guess this should work due to mmtf, but it also feels like there should be different way of doing this, which does not require a List